src/components/Footer/JoinUs/JoinUs.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import './JoinUs.scss';
+import '../Footer.scss';
+
+class JoinUs extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      formError: {
+        email: '',
+      },
+      emailValid: false,
+      formValid: false,
+    };
+  }
+
+  handleInput(e) {
+    e.preventDefault();
+    const name = e.target.name;
+    const value = e.target.value;
+    this.setState({ [name]: value }, () => {
+      this.validateField(value);
+    });
+  }
+
+  validateField(value) {
+    const emailRegEx = /^(\D)+(\w)*((\.(\w)+)?)+@(\D)+(\w)*((\.(\D)+(\w)*)+)?(\.)[a-z]{2,}$/;
+    const fieldValidationError = this.state.formError;
+    let emailValid = this.state.emailValid;
+
+    emailValid = value.match(emailRegEx);
+    fieldValidationError.email = emailValid ? '' : ' is invalid';
+
+    this.setState({ formError: fieldValidationError,
```

Comment:
misaligning here. each object property should be on a new line if there are more then 1 prop